apps/backend/src/modules/onboarding/onboarding.service.ts

```diff
@@ -0,0 +1,77 @@
+import { type Service } from "~/libs/types/types.js";
+
+import {
+	type OnboardingGetAllResponseDto,
+	type OnboardingQuestionDto,
+	type OnboardingQuestionRequestDto,
+} from "./libs/types/types.js";
+import { type OnboardingRepository } from "./onboarding.repository.js";
+import { OnboardingQuestionEntity } from "./onboarding-question.entity.js";
+
+class OnboardingService implements Service {
+	private onboardingRepository: OnboardingRepository;
+
+	public constructor(onboardingRepository: OnboardingRepository) {
+		this.onboardingRepository = onboardingRepository;
+	}
+
+	public async create(
+		payload: OnboardingQuestionRequestDto,
+	): Promise<OnboardingQuestionDto> {
+		const { answers, label } = payload;
+
+		const newQuestionEntity = OnboardingQuestionEntity.initializeNew({
+			answers: answers.map((answer) => ({
+				label: answer.label,
+			})),
```

Comment:
```suggestion
			answers: answers.map((answer) => {
			  return {
				  label: answer.label,
			  }
			}),
```
B27. Fix for other places as well